bin/integration_tests

```diff
@@ -1,34 +1,113 @@
 #!/bin/bash -e
 
+source bin/util
+
+declare -a tests
+no_regen_client=0
+no_rebuild_conjur=0
+
 cleanup() {
   echo "Cleaning up..."
   docker-compose rm --stop --force -v
 }
 
-if [ "$1" == "-d" ]; then
-  DEBUG="true"
-  shift
-fi
+conjur_container_alive(){
+  if [ -z `docker-compose ps -q conjur` ]; then
+    echo 1
+  elif [ -z `docker ps -q --no-trunc | grep $(docker-compose ps -q conjur)` ]; then
+    echo 1
```

Comment:
I don't think these two conditions prove anything different

Reply:
If the first condition returns an empty string then the second condition will Error on the call to `grep`. So We first need to check if the command returns an empty string before checking the second condition.

Reply:
Ok so I think I misunderstood the logic here, but I still think it could be streamlined.

- returning 1 from `conjur_container_alive` when Conjur is not alive seems misleading, hence my other comment
- if `docker-compose ps -q conjur` is not an empty string, then what is the point of using `grep`? Hasn't the first `if` already proved existence? I think `docker-compose ps -q conjur` and `docker ps -q --no-trunc | grep $(docker-compose ps -q conjur)` would just end up returning the same value.

I think this would be more straightforward:
```suggestion
  if [ -n `docker-compose ps -q conjur` ]; then
    echo 1
  else
    echo 0
  fi
```

Reply:
The first command will return a non-empty string if the container is currently on the system. However that doesn't mean that the container is currently alive. The second command lists all currently alive containers and searches for the container we want in that list. The reason I return one from the function is because in bash a return code of 0 means there was no error. So returning 0 in this case would mean that the container is alive.

Reply:
After a second look I recognize that this could be confusing because I do use 0 and 1 as boolean flags for variables. So I switched the function to use 0 and 1 like booleans rather than return codes.